functions/src/CVE/index.ts

```diff
@@ -51,7 +55,7 @@ function getCvesWithBulletinID(id:string,res:any){
     const result = {'CVEs': cves};
     res.send(result);
   }).catch(error => {
-    res.send("error getting CVEs for bulletinID:"+ error);
+    res.status(500).send("error getting CVEs for bulletinID:"+ error);
```

Comment:
Technically our error handling is not correct. Errors 500 mean that something wrong happened with the server, but in case the input data is invalid, we should return status(400).

Reply:
> Technically our error handling is not correct. Errors 500 mean that something wrong happened with the server, but in case the input data is invalid, we should return status(400).

Does that mean I should replace 500 to 400? Is there anything else to add?